mlflow/server/js/src/experiment-tracking/components/experiment-page/components/runs/ExperimentViewRunsTable.tsx

```diff
@@ -370,6 +399,14 @@ export const ExperimentViewRunsTable = React.memo(
                 onGridSizeChanged={({ api }) => gridSizeHandler(api)}
                 onCellMouseOver={cellMouseOverHandler}
                 onCellMouseOut={cellMouseOutHandler}
+                maxBlocksInCache={20} // Increased from 10
```

Comment:
While this is (hopefully) not too much additional memory pressure, has this been confirmed for extremely large collections of runs to not introduce browser stability problems? 

This comment isn't a request to revert this change - just hoping to get a confirmation on any performance validation checks that you did :)